src/content/nl/2020/markup.md

```diff
@@ -0,0 +1,718 @@
+featured_stat_1: 85,73%
+featured_stat_label_1: Percentage pagina's dat het "levende" HTML-doctype gebruikt
+featured_stat_2: 30.073
+featured_stat_label_2: Aantal niet-standaard <code>h7</code>-elementen
+featured_stat_3: 25,24 KB
+featured_stat_label_3: Gewicht van het mediaan document
+---
+
+## Introductie
+
+Het web is gebouwd op HTML. Zonder HTML zijn er geen webpagina's, geen websites, geen webapps. Niets. Er kunnen misschien platte-tekstdocumenten of XML-bomen in een parallel universum zijn die die specifieke uitdaging aangingen. In dit universum vormt HTML de basis van het gebruikersgerichte web. Er zijn veel standaarden waarop het web steunt, maar HTML is zeker een van de belangrijkste.
+
+Hoe gebruiken we HTML dan, hoe groot is onze basis? In het inleidende gedeelte van het [2019 Opmaak-hoofdstuk](../2019/markup#introduction) suggereerde auteur [Brian Kardell](../2019/contributors#bkardell) dat we het lange tijd niet echt hebben geweten. Er waren enkele kleinere monsters. Er was bijvoorbeeld [het onderzoek van Ian Hickson](https://web.archive.org/web/20060203035414/http://code.google.com/webstats/index.html) (een van de moderne HTML-ouders) onder een paar anderen, maar tot vorig jaar misten we veel inzicht in hoe wij als ontwikkelaars, als auteurs, HTML gebruiken. In 2019 hadden we zowel [het werk van Catalin Rosu](https://www.advancedwebranking.com/html/) (een van de co-auteurs van dit hoofdstuk) als de 2019-editie van de Web Almanac om ons weer een beter beeld te geven van HTML in de praktijk.
+
+De analyse van vorig jaar was gebaseerd op 5,8 miljoen pagina's, waarvan 4,4 miljoen op desktop en 5,3 miljoen op mobiel. Dit jaar hebben we 7,5 miljoen pagina's geanalyseerd, waarvan 5,6 miljoen op desktopcomputers en 6,3 miljoen op mobiele apparaten, met behulp van de [laatste gegevens](./methodology#websites) op de websites die gebruikers in 2020 bezoeken. We maken enkele vergelijkingen met vorig jaar, maar net zoals we hebben geprobeerd om aanvullende statistieken te analyseren voor nieuwe inzichten, hebben we ook geprobeerd onze eigen persoonlijkheden en perspectieven door te geven in het hoofdstuk.
+
+<p class="note">
+  In dit Opmaak-hoofdstuk concentreren we ons bijna uitsluitend op HTML, in plaats van SVG of MathML, die ook als opmaaktalen worden beschouwd. Tenzij anders vermeld, verwijzen de statistieken in dit hoofdstuk naar de set mobiele pagina's. Bovendien zijn de gegevens voor alle Web Almanac-hoofdstukken open en beschikbaar. Bekijk <a href="https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/">de resultaten</a> en <a href="https://discuss.httparchive.org/t/2039">deel uw observaties</a> met de community!
+</p>
+
+## Algemeen
+
+In dit gedeelte behandelen we de aspecten van een hoger niveau van HTML, zoals documenttypen, de grootte van documenten en het gebruik van opmerkingen en scripts. "Living HTML" is springlevend!
```

Comment:
Should we do anything about "Living HTML"? Leave as is (but wrap in spans)? OR change to "levende" that you used in featured stat above?

Reply:
Yeah with that one I didn't know if it was a name like Web Almanac but if it's not then it should indeed be levende